pom.xml

```diff
@@ -1648,6 +1652,16 @@
         <artifactId>hbase-zookeeper</artifactId>
         <groupId>org.apache.hbase</groupId>
         <version>${project.version}</version>
+        <exclusions>
+          <exclusion>
+            <groupId>com.google.code.findbugs</groupId>
+            <artifactId>jsr305</artifactId>
```

Comment:
How you conclude these exclusions are necessary? We have a static check someplace?

Reply:
We have an errorprone rule that warns of jsr305 inclusions when release profile is in place.